modelmatrix-core/src/main/scala/com/collective/modelmatrix/ModelMatrix.scala

```diff
@@ -72,16 +72,14 @@ object ModelMatrix extends ModelMatrixUDF {
         build()
   }
 
-  trait PostgresModelMatrixCatalog extends ModelMatrixCatalogAccess {
 
-    protected val driver = slick.driver.PostgresDriver
-    import driver.api._
+  trait DbModelMatrixCatalog extends ModelMatrixCatalogAccess {
```

Comment:
yeah I completely agree, meta data will mostly remain in some type of DB, anyway the ModelMatrixCatalogAccess is also dependent on slick driver so doesn't really make sense to have them apart.